WordPressLoginFlow/src/main/java/org/wordpress/android/login/SignupConfirmationFragment.kt

```diff
@@ -76,7 +78,8 @@ class SignupConfirmationFragment : Fragment() {
             mPhotoUrl = it.getString(ARG_SOCIAL_PHOTO_URL)
             mService = it.getString(ARG_SOCIAL_SERVICE)
         }
-        setHasOptionsMenu(true)
+
+        requireActivity().addMenuProvider(this, viewLifecycleOwner, Lifecycle.State.RESUMED)
```

Comment:
After upgrading the androidX library version, the `setHasOptionsMenu` deprecation error was activated, so I had to migrate this section to make the project green.